Rubberduck.CodeAnalysis/Inspections/Concrete/ImplementedInterfaceMemberInspection.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Rubberduck.Inspections.Abstract;
+using Rubberduck.Inspections.Results;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Resources.Inspections;
+using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Inspections.Inspections.Extensions;
+using Rubberduck.Common;
+
+namespace Rubberduck.Inspections.Concrete
+{
+    /// <summary>
+    /// Identifies implemented members of class modules that are used as interfaces.
+    /// </summary>
+    /// <why>
+    /// Interfaces provide a unified programmatic access to different objects, and therefore are rearly instantiated as concrete objects.
```

Comment:
Typo "rarely"

Reply:
FWIW, the word "rarely" imply there is a legit reason to instantiate an interface. I do not think there is, so I'd go for stronger word like "should not be". Make the user think really carefully about that smell. `</bikeshedding>`

Reply:
IDK, There might be some cases...
[https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-basic-6/aa240846(v=vs.60)](url)

Reply:
Yeah, we actually had a discussion about that in [the chat about this subject](https://chat.stackexchange.com/transcript/14929?m=51277888#51277888). The concern is that without compiler doing any safety checks, an "abstract class" is ultimately doing too much. We want to encourage users from making bad choices - just because VBA allows it does not mean it's a good idea. That said, I'm content to merge this as-is with the "rarely" since this issue is beyond the PR's scope. :)